core/server/worker/src/test/java/alluxio/worker/block/SpaceReserverTest.java

```diff
@@ -75,8 +73,10 @@ public void reserveCorrectAmountsOfSpace() throws Exception {
     Map<String, Long> capacityBytesOnTiers = ImmutableMap.of("MEM", 400L, "HDD", 1000L);
     Mockito.when(storeMeta.getCapacityBytesOnTiers()).thenReturn(capacityBytesOnTiers);
 
+    String tmpFolderPath = mTempFolder.getRoot().getAbsolutePath();
```

Comment:
`String tmpFolderPath = mTempFolder.newFolder().getAbsolutePath();`
The reason you see "java.io.IOException: No such file or directory" is due to the use of `PowerMockRunner`